src/Assets/styleController.ts

```diff
@@ -33,6 +35,7 @@ const themes: IThemes = {
     creamColor: '#F9EFE6',
     terraCottaColor: '#B85A27',
     midBlueColor: '#41528C',
+    brightOrangeColor: '#DE7F43',
```

Comment:
What do you think about removing this js variable since it's not used anywhere?

Reply:
Removed. Using --icon-Color now